src/noise.c

```diff
@@ -69,6 +69,7 @@ int main_noise(int argc, char* argv[])
 
 	float stdev = sqrtf(var);
 
+#pragma omp parallel for
```

Comment:
This makes 'noise' indeterministic, which I think is bad and also defeats the purpose of rinit. We could make it an option (e.g. using #pragma omp for if (flag)) or think of another way to parallelize deterministically